working/augmentation-libraries/feature-specification.md

```diff
@@ -910,7 +959,66 @@ A non-redirecting generative constructor marked `augment` may:
         initializer list containing first the assertions and field initializers
         of the augmented constructor, if any, then the assertions and field
         initializers of the augmenting constructor, and finally any
-        super-initializer of either the augmeted or augmenting constructor.
+        super-initializer of either the augmented or augmenting constructor.
+
+Invoking a non-redirecting generative constructor, with an argument list and a
+new object to initialize, two effects.
+
+*   First it initializes any instance variables in the class which have
+    initializing expressions:
+    *   For each instance variable declared by the class, in source order
```

Comment:
Do we need to define "source order" given that fields might be defined in different parts?

Reply:
I think it's defined somewhere as the "before" order from parts-with-imports. If not, it should be.